### PR TITLE
devdocs: pr title conventions

### PR DESCRIPTION
**Pull request by katexochen**
Signed-off-by: Paul Meyer <49727155+katexochen@users.noreply.github.com>

### Proposed change(s)

- Feel free to discuss

### Checklist

- [x] Update [docs](https://github.com/edgelesssys/constellation/tree/main/docs)
- [x] Add labels (e.g., for changelog category)

### CODE REVIEW

**Review by malt3 — APPROVED** [review]

**Review by daniel-weisse — APPROVED** [review]

File: .github/docs/conventions.md [hunk 1]
```diff
@@ -86,6 +86,34 @@ the `<REPOSITORY>/.vscode/settings.json` repo, so the settings will only affect
...
+
+- the top level directory of the microservice or component, e.g., `joinservice`, `disk-mapper`, `upgrade-agent` but also `docs` and `rfc`
+- in internal, the second level directory
+- `deps` for dependency upgrades
+- `ci` for things that are CI related
+
+and `<title>` is all lower case (except proper names, including acronyms).
+Ticket numbers shouldn't be part of the title.
+
+In case the scope of your PR is to wide, use the alternative format.
```
Comment (leongross) [inline .github/docs/conventions.md:109]:
```suggestion
In case the scope of your PR is too wide, use the alternative format.
```

**Review by leongross — CHANGES_REQUESTED** [review]

**Review by leongross — APPROVED** [review]
lgtm

**Review by thomasten — APPROVED** [review]
❤️

**Review by stdoutput — APPROVED** [review]
👍

File: .github/docs/conventions.md [hunk 2]
```diff
@@ -86,6 +86,34 @@ the `<REPOSITORY>/.vscode/settings.json` repo, so the settings will only affect
...
+
+The PR title should be structured in one of the following ways:
+
+```
+<module>: <title>
+```
+
+Where the `<module>` is
+
+- the top level directory of the microservice or component, e.g., `joinservice`, `disk-mapper`, `upgrade-agent` but also `docs` and `rfc`
```
Comment (Nirusu) [inline .github/docs/conventions.md:101]: But `devdocs` is technically a violation of this rule :P

**Review by Nirusu — APPROVED** [review]

**Review by derpsteb — APPROVED** [review]

**Review by datosh — APPROVED** [review]

**Review by m1ghtym0 — APPROVED** [review]

**Review by 3u13r — APPROVED** [review]

**Pull request merged**